MapboxDirections/MBRouteOptions.swift

```diff
@@ -211,6 +211,13 @@ open class RouteOptions: NSObject {
      */
     open var routeShapeResolution = RouteShapeResolution.low
     
+    /**
+     AttributeOptions for the route. Any combination of `.AttributeOptions` can be specified. 
+     
```

Comment:
Can you elaborate? Do the attributes still work with `.low` resolution? Do the values make any sense?